app/src/main/res/drawable/ic_not_yet_friends.xml

```diff
@@ -0,0 +1,106 @@
+<!--
```

Comment:
Where this is coming from?

Reply:
https://developers.google.com/games/services/checklist